cvat/schema.yml

```diff
@@ -7606,6 +7662,7 @@ components:
           type: string
           format: date-time
           readOnly: true
+          nullable: true
```

Comment:
This seems suspicious. Under what conditions could `joined_date` be unavailable?

Reply:
I don't remember the exact details, but the change is consistent with the current scheme: https://github.com/opencv/cvat/blob/develop/cvat/apps/organizations/models.py#L39. I think this is out of scope for this PR and should be fixed in a separate PR if necessary.

Reply:
Okay.